src/Pages/Dashboard.js

```diff
@@ -13,21 +14,35 @@ import BuyPrompt from '../Components/BuyPrompt'
 import Button from '@material-ui/core/Button';
 
 function DashboardPage(props) {
+  const [userInfo,setUserInfo]= useState({
+    username: props.user.username,
+    loggedIn: props.user.loggedIn,
+  })
```

Comment:
Not sure that we need to keep track of whether or not the user is currently logged in. If the user is not logged in they cannot press any of the buttons to trigger these methods.

Reply:
Good point, also the username will be null if we ever do need to determine if they're logged in or not